File: docs/api/get-products/request.js
// Get-Products RustOpus example

import { writeFile } from "fs/promises";

const AUTHCODE = "AAAAAAAA-0000-0000-0000-BBBBBBBBBBBB";
const OUTFILE = "get-products.xml";

const response = await fetch(`<url-to-rustopus>/get-products?authcode=${AUTHCODE}`);
const text = await response.text();

await writeFile(OUTFILE, text);

File: docs/api/get-products/request.cs
// Get-Products RustOpus example
using System.Net.Http;

var authcode = "AAAAAAAA-0000-0000-0000-BBBBBBBBBBBB";
var outFile = "get-products.xml";

using var client = new HttpClient();
var response = await client.GetStringAsync($"https://<url-to-rustopus>/get-products?authcode={authcode}");
await File.WriteAllTextAsync(outFile, response);
